### PR TITLE
ASI Documentation: Filtering

### PR DESCRIPTION
**Pull request by lenaorobei**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/adobe-stock-integration/issues/1537 and fixes https://github.com/magento/adobe-stock-integration/issues/1536

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs-beta.magento.com/user-guide/cms/media-gallery-asset-management.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cms/media-gallery-asset-management.md [hunk 1]
```diff
@@ -53,6 +53,29 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
     ![]({% link images/images/enhanced-media-gallery/delete-asset-confirmation.png %}){: .zoom}
     _Delete Confirmation_
 
+## Assets filtering
+
+1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.
+
+1. Click <span class="btn">Filters</span>.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:60]: In the screen cap this looks like a tab and not a button.

```suggestion
1. Click the **Filters** tab.
```

File: src/cms/media-gallery-asset-management.md [hunk 2]
```diff
@@ -53,6 +53,29 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
...
+
+    ![]({% link images/images/enhanced-media-gallery/filters.png %}){: .zoom}
+    _Filters_
+
+1. It is possible to filter assets by the entities (Products, Categories, Pages, Blocks, Store Views) where the asset is being used as well as by Content Status. For example, we can easily find assets that are used in the specific category that is enabled.
+    
+    ![]({% link images/images/enhanced-media-gallery/filtered-by-category.png %}){: .zoom}
+    _Filter by Category_
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:68]:
```suggestion
    _Filter for Enabled Assets by Category_
```

File: src/cms/media-gallery-asset-management.md [hunk 3]
```diff
@@ -53,6 +53,29 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
...
+
+## Finding image duplicates
+
+1. Click <span class="btn">Filters</span> and select <span class="btn">Show duplicates</span> checkbox.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:72]:
```suggestion
1. Click the **Filters** tab and select the **Show duplicates** checkbox.
```

File: src/cms/media-gallery-asset-management.md [hunk 4]
```diff
@@ -53,6 +53,29 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
...
+
+1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.
+
+1. Click <span class="btn">Filters</span>.
+
+    ![]({% link images/images/enhanced-media-gallery/filters.png %}){: .zoom}
+    _Filters_
+
+1. It is possible to filter assets by the entities (Products, Categories, Pages, Blocks, Store Views) where the asset is being used as well as by Content Status. For example, we can easily find assets that are used in the specific category that is enabled.
+
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:66]:
```suggestion
    
1. Click <span class="btn">Apply Filters</span> to see the results.

   The filtering in the following example finds assets that are used in a specific category (`cars`) and are enabled.  

```

**Comment by lenaorobei** [comment]
@leslietilling thank you for the review and suggestions.

**Comment by leslietilling** [comment]
running tests

**Comment by leslietilling** [comment]
running tests

**Pull request merged**